src/components/sidenav/sidenavSection/sidenavSection.js

```diff
@@ -26,7 +26,7 @@ export default class SidenavSection {
   }
 
   isActiveState(sref) {
-    return this.$state.current.name === sref;
+    return this.$state.includes(sref);
```

Comment:
Do we also need to change https://github.com/UnityTech/ump-angular-components/blob/2b78bde7b857525ee541438fdee56690f68d82be/src/components/sidenav/sidenavSection/sidenavSection.js#L25 above?

Reply:
don't think we can account for nested states there..in our case when we put IAP Promo links to the operate navigation sidenav, it would be href links - but then after it directs to the IAP app, our implementation of the sidenav would make them sref links so it can account for nested states.